finance/hledger-iadd/Portfile

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           haskell_stack 1.0
+
+github.setup        hpdeifel hledger-iadd 2afeaadbbb29b339b48799ed2bd3112193d64f25
+version             1.3.21
+revision            2
+categories          finance haskell
+depends_run         port:hledger
+
+license             BSD-3-Clause
+maintainers         {fastmail.com:ivofrolov @ivofrolov} \
+                    openmaintainer
+
+description         A terminal UI as drop-in replacement for hledger add
+long_description    ${name} improves in the following ways on hledger's add command: \
+                    \n \
+                    \n* Interactive as-you-type completion for account names and \
+                    descriptions with optional fuzzy matching. \
+                    \n* Integrated calculator. Amounts can be written as simple sums \
+                    with real-time feedback on the result. \
+                    \n* All actions while entering a transaction can be undone. \
+                    \n* Configurable format for date input. Instead of y/m/d it is \
+                    also possible to use other formats like the german d.m.y.
+homepage            https://github.com/hpdeifel/hledger-iadd/
```

Comment:
This line can be dropped because `github.setup` implicitly sets up `homepage`

Reply:
Fixed